src/content/docs/guides/platform-setup/configure-identity-provider.mdx

```diff
@@ -245,7 +245,7 @@ would usually be configured in the external identity provider, in particular
 the issuer url it should use for signing its own JWTs and the allowed redirect
 URLs and audiences of the UI and CLI:
 
-```
+```env
```

Comment:
Shouldn't this be rather `sh`? When I tried to view the syntax highlighting locally, I didn't see anything.